spec/controllers/v0/rated_disabilities_controller_spec.rb

```diff
@@ -33,7 +33,7 @@
         # VCR Cassette should have 3 items in the individual_ratings array, only 2 should
         # be "active"
         parsed_body = JSON.parse(response.body)
-        expect(parsed_body.dig('data', 'attributes', 'individual_ratings').length).to eq(2)
+        expect(parsed_body.dig('data', 'attributes', 'individual_ratings').length).to eq(3)
```

Comment:
This test was affected by the updates to `spec/support/vcr_cassettes/lighthouse/veteran_verification/show/200_response.yml`. Since LH doesn't return inactive ratings anymore, the plan is to remove the check for active ratings from that controller as well. I'm waiting to see what happens after these changes get pushed out and see if I can identify anything else that needs to change before updating the other controller so I can make the changes all at once